xbmc/ServiceManager.cpp

```diff
@@ -138,38 +141,57 @@ bool CServiceManager::StartAudioEngine()
   return m_ActiveAE->Initialize();
 }
 
-bool CServiceManager::Init3()
+bool CServiceManager::InitStageThree()
 {
+  m_peripherals.reset(new PERIPHERALS::CPeripherals(*m_announcementManager));
   m_peripherals->Initialise();
-  m_PVRManager->Init();
```

Comment:
there was a reason why this was in stage three. stage 3 is init after WindowManager is initialized. PVR can have its own startup window, hence need to be in stage 3. on the other hand WindowManager depends on peripherials so peripherials must not be in stage 3